solr/core/src/test/org/apache/solr/handler/component/TestExpandComponent.java

```diff
@@ -234,6 +234,21 @@ private void _testExpand(String group, String floatAppend, String hint) throws E
         "/response/lst[@name='expanded']/result[@name='2"+floatAppend+"']/doc[1]/str[@name='id'][.='8']"
     );
 
+    //Test expand.rows = 0 - no docs only expand count
+    params = new ModifiableSolrParams();
```

Comment:
Same as above so that we can cover more cases

Reply:
done